cmd/kubeadm/app/cmd/alpha/selfhosting.go

```diff
@@ -119,8 +143,15 @@ func getSelfhostingSubCommand() *cobra.Command {
 	// flags bound to the configuration object
 	cmd.Flags().StringVar(&cfg.CertificatesDir, "cert-dir", cfg.CertificatesDir, `The path where certificates are stored`)
 	cmd.Flags().StringVar(&cfgPath, "config", cfgPath, "Path to a kubeadm config file. WARNING: Usage of a configuration file is experimental")
-	cmd.Flags().StringVar(&featureGatesString, "feature-gates", featureGatesString, "A set of key=value pairs that describe feature gates for various features. "+
-		"Options are:\n"+strings.Join(features.KnownFeatures(&features.InitFeatureGates), "\n"))
+
+	cmd.Flags().BoolVarP(
+		&certsInSecrets, "store-certs-in-secrets", "s",
+		false, "Enable storing certs in secrets")
+
+	cmd.Flags().BoolVarP(
+		&forcePivot, "force", "f", false,
+		"Pivot the cluster without prompting for confirmation.",
```

Comment:
please remove the dot at the end.

Reply:
This would be inconsistent with how we did force in `Reset` as well as the other flag messages in Reset.

Reply:
whatever. i'll just delete it so you will merge it.

Reply:
or at least make the flag descriptions consistent in this file.
one has dot, one doesn't.